tools/libs/light/libxl_types.idl

```diff
@@ -802,6 +802,7 @@ libxl_device_pci = Struct("device_pci", [
     ("seize", bool),
     ("rdm_policy", libxl_rdm_reserve_policy),
     ("name", string),
+    ("pcid_domid", integer),
```

Comment:
not sure that this is the best name. `remote_domain` or `backend_domain` will be better.